conf.yaml

```diff
@@ -190,6 +189,12 @@ repository:
                 -
                     action: assign_tag
                     action_value: 'popularity/low'
+                 -
+                    action: remove_tag
+                    action_value: 'popularity/medium
```

Comment:
@erasche I don't think I follow you here. The first link you provided leads to this page and the second checks for the presence of the tag. Do you mean that `self.issue.remove_from_labels(tag_name)` will throw an exception when the tag_name does not exist on the issue?